.github/workflows/build_publish.yml

```diff
@@ -35,3 +46,32 @@ jobs:
           file: package/docker/Dockerfile
           push: true
           tags: bahmni/implementer-interface:${{env.BAHMNI_VERSION}}-${{github.run_number}},bahmni/implementer-interface:latest
```

Comment:
Update `${{env.BAHMNI_VERSION}}-${{github.run_number}} `to `${{env.ARTIFACT_VERSION}}`

Reply:
Okay sure